qiskit/providers/fake_provider/__init__.py

```diff
@@ -0,0 +1,39 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2022.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""
+Utilities for mocking the IBMQ provider, including job responses and backends.
+
+The module includes dummy provider, backends, and jobs.
+The purpose of these classes is to fake backends for testing purposes:
+testing local timeouts, arbitrary responses or behavior, etc.
+
+The mock devices are mainly for testing the compiler.
+"""
+
+from ...test.mock.fake_provider import FakeProvider, FakeLegacyProvider
```

Comment:
When things are far away, it's best to reference them by complete name (e.g. `from qiskit.test.mock.fake_provider`) for legibility.